app/main.py

```diff
@@ -1,3 +1,59 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other) -> Distance:
+        if type(other) == Distance:
+            return Distance(self.km + other.km)
+        else:
+            return Distance(self.km + other)
+
+    def __iadd__(self, other) -> Distance:
+        if type(other) == Distance:
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other) -> Distance:
+        if type(other) == Distance:
+            return Distance(self.km * other.km)
+        else:
+            return Distance(self.km * other)
+
+    def __truediv__(self, other) -> Distance:
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other) -> bool:
+        if type(other) == Distance:
+            return self.km < other.km
+        else:
+            return self.km < other
+
+    def __gt__(self, other) -> bool:
+        return not self.__le__(other)
+
+    def __eq__(self, other) -> bool:
+        return self.km == other
+
+    def __le__(self, other) -> bool:
+        if type(other) == Distance:
+            return self.km <= other.km
+        else:
+            return self.km <= other
+
+    def __ge__(self, other) -> bool:
+        return not self.__lt__(other)
```

Comment:
same here with magic method as in `gt` method

Reply:
And may you answer one question, please: how to name my new commit with changes correctly? I wrote just 'Changed' and don't really understand how to name it better

Reply:
"Fix magic methods usage", "Fix type checks in all methods" for example
You need to describe in the commit what was added, fixed, or changed, so the other people understand what is in this commit.